website/src/utils/yamlDataUtils/create.js

```diff
@@ -20,9 +20,17 @@ export const entity = (dbConn, yamlData) => {
 
     const data = JSON.parse(JSON.stringify(yamlData))
     const steps = ['core_lib', 'entities']
-    const target = getValueAtPath(data, steps)
+    let target = getValueAtPath(data, steps)
+    if(!target){
+        setValueAtPath(data, steps, [])
```

Comment:
will `target = []` will be the same?
same for the others

Reply:
Only for env it will be `{}` otherwise everything is arrays